CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contribution Guidelines
```

Comment:
do we need to or want to enable the "agree to the CLA" github feature as well?

Reply:
@crispywalrus - Yes. This piece still needs to be enabled. I will work on getting that piece approved at the organization level.